src/quacc/utils/files.py

```diff
@@ -289,3 +290,21 @@ def get_uri(directory: str | Path) -> str:
     with contextlib.suppress(socket.gaierror, socket.herror):
         hostname = socket.gethostbyaddr(hostname)[0]
     return f"{hostname}:{fullpath}"
+
+
+def safe_decompress_dir(path: str | Path) -> None:
+    """
+    Recursively decompresses all files in a directory.
+    This is a wrapper around the `decompress_file` function.
+
+
+    Args:
+        path (str | Path): Path to parent directory.
+    """
+    path = Path(path)
+    for parent, _, files in os.walk(path):
+        for f in files:
+            try:
+                decompress_file(Path(parent, f))
+            except FileNotFoundError:
+                logger.warning(f"Cannot find {f} in {parent}. Skipping.")
```

Comment:
I think it'd be okay to just `pass` here since it really is only going to happen in weird concurrent edge cases that the user doesn't care about.